### PR TITLE
Ch refactor dummydata and response 160705643

### PR DESCRIPTION
**Pull request by idmega2000**
#### What does this PR do?
- reorganize dummy data and make response  to have messages
#### Description of Task to be completed?
- [x] reorganize dummy data
- [X] reorganize files and make success responses have messages
- [x] reorganize test files

#### How should this be manually tested?
- clone repo and run npm install etc

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
- story type - chore
- story title - refactor dummy data, output and reorganise files structures
- story id - 160705643
-[story link](https://www.pivotaltracker.com/n/projects/2196606)

### CODE REVIEW

File: server/middleware/checkUserExist.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+     * @returns {object} Returns status code and error messages if error
+     */
+const checkUserExist = (req, res, next) => {
+  const data = req.body.userId;
+  const singleFastFood = fastFoodData.userInformation
+    .find(item => item.userId === data);
+  if (!singleFastFood) {
+    return res.status(404).json({ error: 'User does not Exist' });
+  }
+  /*
```
Comment (dogoyaro) [inline server/middleware/checkUserExist.js]: Please remove commented out code.

File: server/test/order.spec.js [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+        assert.equal(res.body.order.orderId, 'rloahtnnlrv');
+        assert.equal(res.body.order.orderDateTime, '20/09/2018');
+        assert.equal(res.body.order.orderStatus, 'default');
+        done();
+      });
+    });
+  });
+
+
+  describe('when Admin request for A fast food order that does not exist',
```
Comment (dogoyaro) [inline server/test/order.spec.js]: These describe blocks could be simple test cases. Why have you decided to use describe blocks?

File: server/middleware/checkUserExist.js (hunk 1 above)
Comment (idmega2000) [reply server/middleware/checkUserExist.js]: ok i would do that

File: server/test/order.spec.js (hunk 2 above)
Comment (idmega2000) [reply server/test/order.spec.js]: i used describe blocks because i believe it will give better description of individual tests

Comment (dogoyaro) [reply server/test/order.spec.js]: I think you can give good descriptions in the individual test cases. The describe block works best in my opinion to group a bunch of related tests into one logical unit.

Comment (idmega2000) [reply server/test/order.spec.js]: thank you for the feedback. i will implement that now

**Pull request merged**